packages/core/src/blocks/defaultBlockHelpers.ts

```diff
@@ -90,6 +93,27 @@ export const defaultBlockToHTML = <
     );
   }
 
+  // TODO: This is obviously pretty hacky - will need to revisit this when we
+  //  convert default blocks to use the custom block API.
+  if (external) {
+    const dom = renderSpec.dom as HTMLElement;
+
+    if (dom.hasAttribute("data-background-color")) {
+      dom.style.backgroundColor = dom.getAttribute("data-background-color")!;
```

Comment:
nice to have (wouldn't spend too much time on this):

pseudocode that adds this inline style:
```style: {
--blocknote-red: #colorcode;
color: var(blocknote-red);
```
and then detect `var(blocknote-red)` when parsing

This way everything should keep working with color schemes, and external apps would still have the nice color (instead of default css "red")